config-server/src/main/resources/configurations/notification-server.yml

```
@@ -0,0 +1,36 @@
+      username: anup
+      password: anup
+      properties:
+        mail:
+          smtp:
+            trust: "*"
+          auth: true
+          starttls:
+            enable: true
+          connectiontimeout: 5000
+          timeout: 3000
+          writetimeout: 5000
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)